src/components/Score/_Score.scss

```diff
@@ -24,13 +24,13 @@
   }
 
   &.blue-score {
-    background: #5861B4;
-    border: 4px solid rgb(36, 36, 108);
+    background: $blue;
+    border: 4px solid $dark-blue;
```

Comment:
we'll need to check the variables I setup, I gave them pretty descriptive variable names